server/controllers/verify.js

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken';
+import env from 'env2';
+
+env('./config.env');
+
+export default(req, res, next) => {
+  if (req.cookies.AdminToken === undefined) {
+    console.log('not allowed');
+  } else {
+    jwt.verify(req.cookies.AdminToken, process.env.JWT_KEY, (err, decoded) => {
+      if (err) {
+        res.clearCookie('AdminToken');
```

Comment:
(Copying this comment over too 😀)

Here, and line 8 also, you could return send a 401 (unauthorized) response. That'll make things pretty clear in the frontend code what happened (as I mentioned in another comment, on apps I've worked on, we then handle the 401 response by redirecting the user to the login page).